src/pages/steps/AdditionalInformationStep/PriceInformation.tsx

```diff
@@ -170,7 +170,9 @@ const PriceInformation = ({
   }));
 
   const addPriceInfoMutation = useAddOfferPriceInfoMutation({
-    onSuccess: () => setTimeout(() => onSuccessfulChange(), 1000),
+    onSuccess: (data) =>
+      typeof data !== 'undefined' &&
+      setTimeout(() => onSuccessfulChange(), 1000),
```

Comment:
```suggestion
    onSuccess: (data) => {
        if (typeof data === 'undefined') return;
        setTimeout(() => onSuccessfulChange(), 1000);        
    }
```

Reply:
Unless used for conditional templating in the jsx, I prefer a normal if statement over `&&`